examples/src/App.js

```diff
@@ -8,6 +8,7 @@ export default class App extends Component {
     const data = {
       array: [1, 2, 3],
       bool: true,
+      date: Date(),
```

Comment:
`Date()` returns a string object, not a date object. I'd use `new Date() instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date